includes/level-settings.php

```diff
@@ -76,10 +76,10 @@ function pmprodon_pmpro_save_membership_level( $level_id ) {
 	} else {
 		$donations_only = 0;
 	}
-	$min_price       = preg_replace( '[^0-9\.]', '', $_REQUEST['donation_min_price'] );
-	$max_price       = preg_replace( '[^0-9\.]', '', $_REQUEST['donation_max_price'] );
+	$min_price       = $_REQUEST['donation_min_price'];
+	$max_price       = $_REQUEST['donation_max_price'];

```

Comment:
We need sanitization here. Even if these fields are `number` HTML fields, the HTML source can still be edited to make them `text` and non-numeric data could still be submitted. It is not a likely scenario, but we should still be protecting against that